src/net/http.rs

```diff
@@ -929,6 +930,9 @@ impl HttpResponsePreamble {
         fd.write_all(format!("{} {}\r\n", status_code, reason).as_bytes()).map_err(net_error::WriteError)?;
         fd.write_all("Server: stacks/2.0\r\nDate: ".as_bytes()).map_err(net_error::WriteError)?;
         fd.write_all(rfc7231_now().as_bytes()).map_err(net_error::WriteError)?;
+        fd.write_all("\r\nAccess-Control-Allow-Origin: *".as_bytes()).map_err(net_error::WriteError)?;
```

Comment:
I don't think these headers belong here -- we don't need to send them with each response.  You should implement a CORS headers callback to be fed into `new_serialized()` instead.

Reply:
Why don't we need them for every request? We need them at least for each RPC API.

Reply:
Isn't that handled with the `OPTIONS` endpoint?  i.e. the client does `OPTIONS ${path}`, gets the CORS headers, and then does `GET ${path}` or `POST ${path}`?